File: 코딩테스트 입문/Javascript/홀수 vs 짝수.js
function solution(num_list) {
    let answer = 0;
    let [odd,even] = [0,0];
    
    num_list.forEach((v,i)=>{
        if(i%2===0){
            even+=v;
        }
        else{
            odd+=v;
        }
    })
    
    answer = (even > odd) ? even : odd;
    
    return answer;
}

File: 코딩테스트 입문/Python/홀수 vs 짝수.py
def solution(num_list):    
    answer = 0
    oddSum = 0
    evenSum = 0
    
    for index, value in enumerate(num_list):
        if index%2==0:
            evenSum += value
        else:
            oddSum += value
    
    answer = max(evenSum, oddSum)
    
    return answer